modules/reference/pages/rpk/rpk-security/rpk-security-role.adoc

```diff
@@ -0,0 +1,36 @@
+= rpk security role
+
+Manage Redpanda roles.
+
+== Usage
+
+[,bash]
+----
+rpk security role [command] [flags]
+----
+
+== Aliases
+
+[,bash]
+----
+role, access, roles
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|--format |string |Output format. Possible values: `json`, `yaml`, `text`, `wide`, `help`. Default: `text`.
+
+|-h, --help |- |Help for role.
+|--config |string |Redpanda or `rpk` config file; default search paths are `~/.config/rpk/rpk.yaml`, `$PWD`, and `/etc/redpanda/redpanda.yaml`.
+|--config |string |Redpanda or rpk config file; default search paths are `/var/lib/redpanda/.config/rpk/rpk.yaml`, `$PWD/redpanda.yaml`, and `/etc/redpanda/redpanda.yaml`.
```

Comment:
This is kind of confusing--the same option returns different paths? Why is it here twice?

Reply:
This might be an error from the script :thinking: The output I see in rpk is:

```
Flags:
      --format string   Output format (json,yaml,text,wide,help) (default "text")
  -h, --help            Help for role

Global Flags:
      --config string            Redpanda or rpk config file; default search paths are "/home/rvasquez/.config/rpk/rpk.yaml", $PWD/redpanda.yaml, and
                                 /etc/redpanda/redpanda.yaml
  -X, --config-opt stringArray   Override rpk configuration settings; '-X help' for detail or '-X list' for terser detail
      --profile string           rpk profile to use
```

Reply:
def an error. Will look into it

Reply:
fixed